### ISSUE TITLE
usgs module

### ISSUE THREAD

**Issue by kapadia** [developmentseed/modis-metadata#1]
Hey @drewbo - could the [USGS module](https://github.com/mapbox/usgs) serve your purposes for gathering MODIS metadata? As an example:

```
usgs search --node EE MODIS_MYD09A1 --start-date 20150501 --end-date 20150531 --geojson | gist -f modis-myd09a1.geojson
```

**Comment by drewbo** [comment]
@kapadia Looks awesome (sorry for slow response). Do you know how to obtain the correct account privileges?

```
Authentication Failed: User does not have Inventory Service access. 
```

**Comment by kapadia** [comment]
@drewbo metadata search requires no authentication. is that output from `usgs login`?

**Comment by drewbo** [comment]
Yeah, I'll search without logging in; do you know what portions do require the login?

**Comment by kapadia** [comment]
Obtaining direct download urls requires authentication with an account that has elevated privileges granted by EROS. Getting permissions from EROS is tricky business; we went through this effort when getting Landsat 8 on AWS.
